### PR TITLE
Issue65

### PR DESCRIPTION
**Pull request by himanshugoyal1065**
Made changes regarding the issue 65

### CODE REVIEW

**Comment by himanshugoyal1065** [comment]
@marcinadd The build is failing for the test suite, ProjectServiceTests, which passes locally. I don't know how to recreate the issue locally and hence fix it. 
I request you to take some time and have a look at it. Please let me know if you need some help.

File: src/main/resources/application-development.properties [hunk 1]
```diff
@@ -1,10 +1,11 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/projecty
+spring.datasource.url=jdbc:mysql://localhost:3306/projecty?createDatabaseIfNotExist=true
 spring.datasource.username=root
 spring.datasource.password=password
 spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL55Dialect
```
Comment (marcinadd) [inline src/main/resources/application-development.properties]: In README.md requirements are to use MySQL8 so you shouldn't use MySQL55Dialect

Comment (himanshugoyal1065) [reply src/main/resources/application-development.properties]: Changed the value of the dialect to org.hibernate.dialect.MySQL8Dialect

**Review by marcinadd — COMMENTED** [review]
Ok. I will look at your code tomorrow and I will try to fix tests.

**Comment by marcinadd** [comment]
Sorry for the delay. I hope I will look at it tomorrow.

**Review by marcinadd — APPROVED** [review]
Ok I squashed your commits into one and fixed tests in PR #69. 
Your code will be merged.
Thanks for your contribution.

**Pull request closed without merging**